index.html

```diff
@@ -62,23 +62,37 @@ <h1>Hello, world!</h1>
     </div>
 
     <div class="container">
+      <div class="poem-container">
```

Comment:
Creating the poem-container makes it easy to style multiple poems especially in this project where more could added.